src/benchmarklib/benchmark_utils.cpp

```diff
@@ -330,6 +331,43 @@ nlohmann::json EncodingConfig::to_json() const {
   return json;
 }
 
+void BenchmarkTableEncoder::encode(const std::string& table_name, std::shared_ptr<Table> table,
+                                   const EncodingConfig& config) {
+  const auto& type_mapping = config.type_encoding_mapping;
+  const auto& custom_mapping = config.custom_encoding_mapping;
+
+  const auto& column_mapping_it = custom_mapping.find(table_name);
+  const auto table_has_custom_encoding = column_mapping_it != custom_mapping.end();
+
+  ChunkEncodingSpec chunk_spec;
+
+  for (ColumnID column_id{0}; column_id < table->column_count(); ++column_id) {
+    if (table_has_custom_encoding) {
+      const auto& column_name = table->column_name(column_id);
+      const auto& column_mapping = column_mapping_it->second;
```

Comment:
That could be outside the loop - if you want to. And the name could be more descriptive, e.g., "encoding_by_column_name"

Reply:
alright, name change will be sufficient